CBinding/CMake/CMakeArgument.cs

```diff
@@ -0,0 +1,89 @@
+﻿//
+// CMakeArgument.cs
+//
+// Author:
+//       Elsayed Awdallah <comando4ever@gmail.com>
+//
+// Copyright (c) 2015 Xamarin Inc. (http://xamarin.com)
```

Comment:
Copyright actually remains with you. Xamarin doesn't own this code, we just require it's under the MIT license :)